task_spec/spec/acceptance/nodesets/docker/rocky-8.yml

```diff
@@ -1,14 +1,14 @@
 HOSTS:
```

Comment:
We should still keep the old `centos-7` file, otherwise docker PR tests will fail

Reply:
Fixed sorry